app/migrations/init_db.py

```diff
@@ -311,6 +311,15 @@ def init_db():
                     unique_column text not NULL 
                     )
                     ''')
+    cur.execute('''
+                    CREATE TABLE IF NOT EXISTS alias
+                    (
+                    identifier text  NOT NULL,
+                    wallet_address text NOT NULL,
+                    address text NOT NULL,
```

Comment:
what is `address` used for?

Reply:
It is contract address. It is the pattern that sc private state will have address (contract address) and other unique column so that all entries can be uniquely identified with that combination (cause two contracts can have same value).